prov/efa/src/rdm/efa_rdm_msg.c

```diff
@@ -172,7 +172,8 @@ ssize_t efa_rdm_msg_post_rtm(struct efa_rdm_ep *ep, struct efa_rdm_ope *txe, int
 
 	if (rtm_type < EFA_RDM_EXTRA_REQ_PKT_BEGIN) {
 		/* rtm requires only baseline feature, which peer should always support. */
-		return efa_rdm_ope_post_send(txe, rtm_type);
```

Comment:
I could return the result of`1efa_rdm_ope_post_send` here because the error handling is only for long read protocl which is not selected here